### ISSUE TITLE
Feature request : x4 (drawing) model

### ISSUE THREAD

**Issue by RobbWatershed** [Lucchetto/SuperImage#50]
Would it be possible to integrate a lighter and faster "realesrgan-x2plus-anime" model (which would be labeled as "x4 (drawing)" ?

**Comment by Lucchetto** [comment]
Yes, eventually I'll make a x2 anime model or look for a pretrained one
